docs/sources/k6-studio/set-up/install.md

```diff
@@ -6,24 +6,47 @@ weight: 100
 
 # Install k6 Studio
 
-You can install k6 Studio on Windows and macOS.
+You can install k6 Studio on Windows, macOS, and Linux.
 
 ## Before you begin
 
 - Install [Google Chrome](https://www.google.com/chrome/) on your computer.
 
-## Install k6 Studio
+## Install k6 Studio on Mac
 
-To install k6 Studio:
+To install k6 Studio on macOS:
 
-- Go to the [Releases page](https://github.com/grafana/k6-studio/releases) in the k6 Studio GitHub repository.
-- Download the executable file for your system.
-  - **For Windows**: Click on **k6.Studio-{VERSION}-Setup.exe**.
-  - **For macOS**: Click on **k6.Studio-{VERSION}-arm64.dmg**.
-- After the download finishes, open the executable file.
+1. Go to the [Releases page](https://github.com/grafana/k6-studio/releases) in the k6 Studio GitHub repository.
+1. Download the executable file for macOS:
+   - If your Mac has an Apple silicon processor, download **k6.Studio-{VERSION}-arm64.dmg**.
+   - If your Mac has an Intel silicon processor, download **k6.Studio-{VERSION}-x64.dmg**.
+1. After the download finishes, open the executable file.
+1. When prompted, move the k6 Studio application file to the Applications folder. This ensure updates can be installed correctly.
 
-{{< admonition type="note" >}}
+## Install k6 Studio on Windows
 
-For macOS users, make sure that you install k6 Studio in your Applications folder. If you install it in your Downloads folder, the application won't work correctly.
+To install k6 Studio on Windows:
 
-{{< /admonition >}}
+1. Go to the [Releases page](https://github.com/grafana/k6-studio/releases) in the k6 Studio GitHub repository.
+1. Download the executable file for Windows: **k6.Studio-{VERSION}-Setup.exe**.
+1. After the download finishes, open the executable file.
+
+## Install k6 Studio on Linux
+
+To install k6 Studio on Linux:
+
+1. Go to the [Releases page](https://github.com/grafana/k6-studio/releases) in the k6 Studio GitHub repository.
+1. Download the executable file for Linux:
+   - If you're using a Debian-based system, download **k6.Studio-{VERSION}-amd64.deb**.
+   - If you're using a Red Hat-based system, download **k6.Studio-{VERSION}-x86_64.rpm**.
+1. After the download finishes, open your terminal and run:
+
+```bash
+sudo dpkg -i <K6_STUDIO_EXECUTABLE>
+```
+
+## Update k6 Studio
+
+When using k6 Studio on Mac or Windows, k6 Studio automatically updates whenever a new version is released.
+
+For Linux systems, you have to manually update the application.
```

Comment:
@Llandy3d are there any specific instructions for how to update on Linux? 🤓

Reply:
I can only assume that it will be the same command used to install the first time, for example on Ubuntu `sudo dpkg -i <k6-studio-installer>`

Reply:
Is there any way we could test/confirm this? 🤓

Reply:
AFAIK to update the package you'd need to install it again, the way @Llandy3d suggested.

Reply:
We can confirm tomorrow when we try the new release 🤔